Automattic/jetpack
# Admin page: remove alt attribute from decorative icon

## Proposed changes:

The Alt value isn't necessary here, since the icon doesn't bring any additional info on the page for folks using screen readers.

This has the added benefit of getting rid of a React warning:

```
Warning: Failed prop type: Invalid prop `alt` of type `object` supplied to `PlanIcon`, expected `string`.
PlanIcon@wp-content/plugins/jetpack/_inc/build/admin.js?ver=15a830895a6f6d1e3c39:44686:1
MyPlanCard@wp-content/plugins/jetpack/_inc/build/admin.js?ver=15a830895a6f6d1e3c39:51709:20
```

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site that's connected to WordPress.com.
* Via Store admin, add a legacy Jetpack Backup Daily plan to your site.
* Open your JS console.
* Go to Jetpack > Dashboard > My Plan
    * You should not see the React warning from above.
    * The `alt` attribute from the plan icon should have an empty value (it previously had an invalid object).

<img width="834" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/8f08bf53-7fc0-4c4e-8377-3a1320f56877">
